corehq/motech/openmrs/repeater_helpers.py

```diff
@@ -716,6 +721,17 @@ def get_relevant_case_updates_from_form_json(domain, form_json, case_types, extr
     return result
 
 
+def get_unknown_encounter_role(requests):
+    """
+    Return "Unknown" encounter role for legacy providers with no
+    encounter role set
+    """
+    response_json = requests.get('/ws/rest/v1/encounterrole').json()
+    for encounter_role in response_json['results']:
+        if encounter_role['display'] == 'Unknown':
+            return encounter_role
```

Comment:
nit: if no role is found meeting the condition, this loop will complete and the function will implicitly `return None`. It's not clear if that would be a bug or the desired outcome. Explicitly returning a value or raising an exception after the loop is suggested.